BenadrylAutomation/src/main/java/com/jnj/base/BaseClass.java

```diff
@@ -63,15 +58,11 @@ public void loadConfig() throws IOException {
 
 	public static void launchApplication() {
 		String browserName = prop.getProperty("browserName");
-		runOn = prop.getProperty("runOn");
-		String headlessChrome = prop.getProperty("headlessChrome");
+		String runOn = prop.getProperty("runOn");
 		if (browserName.contains("Chrome")) {
 			ChromeOptions options = new ChromeOptions();
```

Comment:
@zahinSJI why did you removed code for headless chrome??

Reply:
The code is still there in my code
![image](https://github.com/sji-autobots/BenadrylAutomation/assets/107922907/a762b1fe-efcb-4475-a4a0-ba5b5223baf5)